pulumi/grapl/__main__.py

```diff
@@ -301,13 +300,6 @@ def main() -> None:
         opts=pulumi.ResourceOptions(provider=consul_provider),
     )
 
-    ConsulServiceDefault(
```

Comment:
There's a possibility that we may need to add this back temporarily so we can delete the service first and then delete this. No action needed now, but that's what we'll have to do if we run into problems with pulumi